assets/prototype/application/ui/components/layout/editor-modal/useEditorModalState/index.ts

```diff
@@ -2,40 +2,36 @@
  * External dependencies
  */
 import { useReducer } from 'react';
-import { assoc, dropLast, last } from 'ramda';
+import { dropLast, last } from 'ramda';
 
 enum ActionType {
 	CLOSE_ALL = 'CLOSE_ALL',
 	CLOSE_EDITOR = 'CLOSE_EDITOR',
 	OPEN_EDITOR = 'OPEN_EDITOR',
 }
 
-type State = {
-	stack: string[];
-};
+type State = string[];
 
 type Action = {
 	editorId?: string;
 	type: ActionType;
 };
 
 const reducer = (state: State, action: Action) => {
-	const { stack } = state;
 	const { editorId } = action;
 
 	switch (action.type) {
 		case ActionType.CLOSE_EDITOR:
-			if (editorId === last(stack)) {
-				const stackWithoutLastItem = dropLast(1, stack);
-				return { ...state, stack: stackWithoutLastItem };
+			if (editorId === last(state)) {
+				const stackWithoutLastItem = dropLast(1, state);
+				return stackWithoutLastItem;
```

Comment:
maybe just return directly and use a code comment to clarify what's happening, ie:

```js
// return stack after removing the last editor added (top of stack)
return dropLast(1, state);
```

Reply:
I like the comment more than `stackWithoutLastItem`.
Thx.
Updated: https://github.com/eventespresso/event-espresso-core/pull/2171/commits/b0b07f5dce7f2dab3c8141220b25887fee8c8448